### PR TITLE
docs: update src14 storage recommendation and general typos

### LINKED ISSUE #133: src14: update namespace recommendation docs to match new slot calculation

### LINKED ISSUE #134: chore: fix typos

### PR DESCRIPTION
**Pull request by K1-R1**
## Type of change

- Documentation

## Changes

The following changes have been made:

- SRC14: as of `forc` > v0.61.0 storage slots now are calculated differently, SRC14 has been updated to recommend namespacing all non-standardized storage variables under the `SRC14` namespace.
- General markdown file typos have been fixed.
- General markdown file formatting.

## Related Issues

Closes #133 
Closes #134 

## Checklist

- [x] I have linked to any relevant issues.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
   - [x] If my change requires substantial documentation changes, I have [requested support from the DevRel team](https://github.com/FuelLabs/devrel-requests/issues/new/choose)
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [x] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [x] I have requested a review from the relevant team or maintainers.
- [x] I have updated the changelog to reflect the changes on this PR.

### CODE REVIEW

**Comment by K1-R1** [comment]
To be merged after #135

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Example of the SRC-3 implementation where a contract mints multiple assets with
 
 #### SRC-5; Ownership Examples
 
-##### - [Uninitalized](./examples/src5-ownership/uninitialized_example/src/uninitialized_example.sw)
+##### - [Uninitialized](./examples/src5-ownership/uninitialized_example/src/uninitialized_example.sw)
```
Comment (bitzoic) [inline README.md:112]: If this didn't get caught by CI it means that we have a misspelled word in the custom spell check word list

File: docs/src/src-10-native-bridge.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ A standard interface for bridges intends to provide a safe and efficient bridge
 
 ## Prior Art
 
-The standard is centered on Fuel’s [Bridge Architecture](https://github.com/FuelLabs/fuel-bridge/blob/main/docs/ARCHITECTURE.md). Fuel's bridge system is built on a message protocol that allows to send (and receive) messages between entities located in two different blockchains.
+The standard is centred on Fuel’s [Bridge Architecture](https://github.com/FuelLabs/fuel-bridge/blob/main/docs/ARCHITECTURE.md). Fuel's bridge system is built on a message protocol that allows to send (and receive) messages between entities located in two different blockchains.
```
Comment (bitzoic) [inline docs/src/src-10-native-bridge.md:11]: British vs American English

File: docs/src/src-14-simple-upgradeable-proxies.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ Proxy designs fall into three essential categories:
 
 This document falls in the second category. We want to standardize the implementation of simple upgradeable pass-through contracts.
 
-The FuelVM provides an `LDC` instruction that is used by Sway's `std::execution::run_external` to provide a similar behavior to EVM's `delegatecall` and execute instructions from another contract while retaining one's own storage context. This is the intended means of implementation of this standard.
+The FuelVM provides an `LDC` instruction that is used by Sway's `std::execution::run_external` to provide a similar behaviour to EVM's `delegatecall` and execute instructions from another contract while retaining one's own storage context. This is the intended means of implementation of this standard.
```
Comment (bitzoic) [inline docs/src/src-14-simple-upgradeable-proxies.md:21]: British vs American English

File: docs/src/src-14-simple-upgradeable-proxies.md [hunk 4]
```diff
@@ -18,14 +18,14 @@ Proxy designs fall into three essential categories:
...
 
 ## Specification
 
-### Required Behavior
+### Required Behaviour
```
Comment (bitzoic) [inline docs/src/src-14-simple-upgradeable-proxies.md:25]: British vs American English

File: docs/src/src-2-inline-documentation.md [hunk 5]
```diff
@@ -300,7 +300,7 @@ This standard will improve security by providing developers with relevant inform
 
 ### Function Example
 
-```sway
+````sway
```
Comment (bitzoic) [inline docs/src/src-2-inline-documentation.md:303]: Should this have 4 ticks?

File: docs/src/src-2-inline-documentation.md [hunk 6]
```diff
@@ -93,14 +93,14 @@ This section has a `h1` header.
 This section provides an example of the use of the function. This section is not required to follow the SRC-2 standard however encouraged for auxiliary and library functions.
 Example:
 
-```sway
+````sway
```
Comment (bitzoic) [inline docs/src/src-2-inline-documentation.md:96]: 4 ticks?

File: CHANGELOG.md [hunk 7]
```diff
@@ -17,7 +17,6 @@ Description of the upcoming release here.
 ### Changed Unreleased
 
 - [#135](https://github.com/FuelLabs/sway-standards/pull/135) Updates standards, examples and CI to latest forc 0.63.3.
-- Something changed here 2
```
Comment (SwayStar123) [inline CHANGELOG.md:20]:
```suggestion
[#136](https://github.com/FuelLabs/sway-standards/pull/136) Fix typos, update src14 storage recommendation, and improve formatting
```

**Review by SwayStar123 — DISMISSED** [review]

**Review by bitzoic — APPROVED** [review]

**Review by SwayStar123 — APPROVED** [review]

**Pull request merged**